cpp/ql/lib/semmle/code/cpp/ir/implementation/aliased_ssa/internal/AliasAnalysis.qll

```diff
@@ -338,6 +338,57 @@ private predicate resultEscapesNonReturn(Instruction instr) {
   not instr.isResultModeled()
 }
 
+/** Holds if `operand` may (transitively) flow to an `AddressOperand` whose use is unmodeled. */
+private predicate consumedAsAddressOperand(Operand operand) {
+  operand instanceof AddressOperand and
+  exists(Instruction use | not use.isResultModeled())
```

Comment:
I'm expecting some kind of relation between the operand and the use here, but there seems to be none?

Reply:
Oops. Yes, there definitely should be. The operand should be an operand of `use`. Good catch!

Reply:
Fixed in a5efe9fa099e4a64f775c9cb45769d7a403d1a17. Turns out the "unmodeled" part was an incorrect optimization. I've removed that part now.